pkg/crc/ssh/client.go

```diff
@@ -105,6 +106,10 @@ func (client *NativeClient) Run(command string) ([]byte, []byte, error) {
 	session.Stderr = &stderr
 
 	err = session.Run(command)
+	if err != nil {
```

Comment:
Can we/do we need to differentiate between "The command that we ran returned an error" and "There was an error in the ssh communication"?

Reply:
@cfergeau we do need to catch specific ssh type error and see if this can be used.